libr/core/cconfig.c

```diff
@@ -2565,6 +2565,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF ("http.maxport", "9999", "Last HTTP server port");
 	SETPREF ("http.ui", "m", "Default webui (enyo, m, p, t)");
 	SETPREF ("http.sandbox", "true", "Sandbox the HTTP server");
+	SETPREF ("http.no_pipe", "false", "Workaround for slurp error without sandbox");
```

Comment:
config vars dont have lowerdash

Reply:
this should work always, dont add extra config vars, just fix the bug

Reply:
What's the meaning of command piping? Under the debug tmp file doesn't contain any command. Looks like r_cmd_flush does nothing. I wasn't writing that code. Need some time to understand.